backend/settings.py

```diff
@@ -230,6 +230,8 @@
     CSRF_TRUSTED_ORIGINS = os.getenv("CSRF_TRUSTED_ORIGINS", [])
     SESSION_COOKIE_SECURE = True
     SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+    # To set the correct hostname for DRF when running https
+    USE_X_FORWARDED_HOST = True
```

Comment:
A more general comment may be appropriate, like "# Enables proxies that set headers".